Makefile.am

```diff
@@ -83,15 +83,16 @@ precompile:
 	$(bindir)/lepton-schematic
 
 
-uninstall-local:
-	rm -fr $(datadir)/lepton-eda/ccache
```

Comment:
Is this change intended? I'm sorry for my ignorance wrt what the code used for.

Reply:
this is superseded by `rm -fr $(pkgdatadir)`
`datadir = share`
`pkgdatadir = share/lepton-eda`

Reply:
OK.